### PR TITLE
RELATED: RAIL-3725 Drill setup using catalog export docs

### PR DESCRIPTION
**Pull request by plavemar**
- Documentation on how to disable drilling from axis labels

JIRA: RAIL-3725

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/929/ : SUCCESS in 1m 07s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/617/ : SUCCESS in 7s

File: docs/15_props__chart_config.md [hunk 1]
```diff
@@ -464,3 +465,7 @@ A headline chart with `config.enableCompactSize` set to `true` adjusts its conte
 ### Responsive legend
 
 For configuring a responsive legend, see [Change legend properties](#change-legend-properties).
+
+### Drilling on axis labels
+
+Drilling on axis labels can be disabled setting `forceDisableDrillOnAxes` property.
```
Comment (no23reason) [inline docs/15_props__chart_config.md:471]: Drilling on axis labels can be disabled by setting `forceDisableDrillOnAxes` property to `true`.

**Review by andriichumak — APPROVED** [review]

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/930/ : SUCCESS in 1m 07s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/618/ : SUCCESS in 8s

File: docs/15_props__chart_config.md [hunk 2]
```diff
@@ -464,3 +465,7 @@ A headline chart with `config.enableCompactSize` set to `true` adjusts its conte
 ### Responsive legend
 
 For configuring a responsive legend, see [Change legend properties](#change-legend-properties).
+
+### Drilling on axis labels
```
Comment (evgeniabogranskaya) [inline docs/15_props__chart_config.md:469]: ### Disabled drilling on axis labels

File: docs/15_props__chart_config.md [hunk 3]
```diff
@@ -464,3 +465,7 @@ A headline chart with `config.enableCompactSize` set to `true` adjusts its conte
...
+
+Drilling on axis labels can be disabled by setting `forceDisableDrillOnAxes` property to `true`.
```
Comment (evgeniabogranskaya) [inline docs/15_props__chart_config.md:471]: To disable drilling on axis labels, set `forceDisableDrillOnAxes` to `true`.

Comment (evgeniabogranskaya) [reply docs/15_props__chart_config.md:471]: Does this setting disables drilling on both axes?

Comment (plavemar) [reply docs/15_props__chart_config.md:471]: yes, it disables drilling from labels completely.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/931/ : SUCCESS in 1m 31s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/619/ : SUCCESS in 8s

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-gate-PR476

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/932/ : SUCCESS in 2m 26s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-fossa-scanning-zuul-el7/180/ : SUCCESS in 41s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sonarqube-scan-zuul-gate/75/ : SUCCESS in 57s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/270/ : SUCCESS in 22s

**Pull request merged**